tasmota/tasmota_template_ESP32.h

```diff
@@ -222,7 +224,9 @@ const char kSensorNames[] PROGMEM =
   D_SENSOR_WINDMETER_SPEED "|" D_SENSOR_BUTTON "_tc|"
   D_SENSOR_BL0940_RX "|"
   D_SENSOR_TCP_TXD "|" D_SENSOR_TCP_RXD "|"
-  D_SENSOR_ETH_PHY_POWER "|" D_SENSOR_ETH_PHY_MDC "|" D_SENSOR_ETH_PHY_MDIO
+  D_SENSOR_ETH_PHY_POWER "|" D_SENSOR_ETH_PHY_MDC "|" D_SENSOR_ETH_PHY_MDIO "|"
+  D_SENSOR_TCP_TXD "|" D_SENSOR_TCP_RXD "|"
```

Comment:
Redundant. Should not be here. See two lines above.

Reply:
done